From 73998e9600d82cb45ab115de7e24585436f5b519 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Wed, 4 Dec 2024 13:43:31 +0000
Subject: [PATCH] arm64: dts: broadcom: Fixup HVS address for downstream
 size-cells

Downstream is setting #size-cells = <1> cf <2> for upstream.
Update the HVS regs property to match.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm64/boot/dts/broadcom/bcm2712.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/arch/arm64/boot/dts/broadcom/bcm2712.dtsi
+++ b/arch/arm64/boot/dts/broadcom/bcm2712.dtsi
@@ -463,10 +463,10 @@
 
 	hvs: hvs@107c580000 {
 		compatible = "brcm,bcm2712-hvs";
-		reg = <0x10 0x7c580000 0x0 0x1a000>;
+		reg = <0x10 0x7c580000 0x1a000>;
 		interrupt-parent = <&disp_intr>;
 		interrupts = <2>, <9>, <16>;
 		interrupt-names = "ch0-eof", "ch1-eof", "ch2-eof";
-		//iommus = <&iommu4>;
+		iommus = <&iommu4>;
 	};
 };
